examples/immersive-typography/basic/example.js

```diff
@@ -0,0 +1,58 @@
+let angle = 0;
+let x = 0;
+let speed = 0.01;
+
+let graphics;
+
+function preload() {
+  createVRCanvas();
+}
+
+function setup() {
+  setVRBackgroundColor(200,200,200);
+ 
+  //createGraphics MUST be in setup, other attributes can go in draw()
+  graphics = createGraphics(600,600);
+}
+
+function draw() {
+  setViewerPosition(0, 0, 600);
+  
+  //must put "name." or else texture attributes wont work
```

Comment:
This comment is a bit confusing. I get that you are explaining how graphics work but 'name.' isn't the clearest.